gabbi/runner.py

```diff
@@ -89,22 +93,28 @@ def run():
         failure = not success
     else:
         for input_file in input_files:
+            name = os.path.splitext(os.path.basename(input_file))[0]
             with open(input_file, 'r') as fh:
                 data_dir = os.path.dirname(input_file)
                 success = run_suite(fh, handler_objects, host, port,
                                     prefix, force_ssl, failfast,
                                     data_dir=data_dir,
-                                    verbosity=verbosity)
+                                    verbosity=verbosity, name=name)
+            if not success:
```

Comment:
It was intentional pattern replication because sadly success and failure aren't meaning the same things. success is a value for right now, failure is a value for since we started.